state/userAccount.go

```diff
@@ -31,10 +33,21 @@ func NewEmptyUserAccount() *userAccount {
 }
 
 // NewUserAccount creates new simple account wrapper for an AccountContainer (that has just been initialized)
-func NewUserAccount(address []byte, hasher hashing.Hasher, marshaller marshal.Marshalizer) (*userAccount, error) {
+func NewUserAccount(
```

Comment:
might update all the comments with this format:
`// New* creates new simple account wrapper for an AccountContainer (that has just been initialized)`
since I guess that `AccountContainer` does not exist anymore

also, `new` -> `a new`

Reply:
Done.

Reply:
still not done for peer account